src/cli/data.py

```diff
@@ -220,12 +218,12 @@ def list(self):
             pdata = dict(Name=[], Type=[], ID=[])
 
             for item in rows:
-                pdata["Name"].append(item.name)
+                pdata["Name"].append(getattr(item, "name", None))
+
```

Comment:
this construction (you'll see this everywhere) works even when there is no attribute named item